README.md

```diff
@@ -6,7 +6,7 @@ Tesla Middleware for Spandex that supports distributed tracing
 ```elixir
 def deps do
   [
-    {:spandex_tesla_distributed, github: "smartrent/spandex_tesla_distributed", branch: "master"}
+    {:spandex_tesla_distributed, "~> 0.1.0"}
```

Comment:
Think we need to still specify the organization here perhaps

Reply:
This isn't a published package.   We shouldn't be changing that.

Reply:
How do we feel about specifying a tag instead of a branch so we can control the updates elsewhere?

```
:spandex_tesla_distributed, github: "smartrent/spandex_tesla_distributed", tag: "0.1.0"
```

Reply:
@DevBaca the mix.lock file should keep us locked in on a specific commit, unless we want to specifically pin this version I think we leave it pointing at the main branch.